modular_zubbers/code/modules/reagents/chemistry/reagents/other.dm

```diff
@@ -0,0 +1,9 @@
+
+//Splash-based recipe to turn improvised gauze into sterile gauze.
+/datum/reagent/space_cleaner/sterilizine/expose_obj(obj/O, reac_volume)
+	. = ..()
+	if(istype(O, /obj/item/stack/medical/gauze/improvised))
+		var/obj/item/stack/medical/gauze/G = O
+		reac_volume = min((reac_volume/10), G.amount)
+		new /obj/item/stack/medical/gauze(get_turf(G), reac_volume)
+		G.use(reac_volume)
```

Comment:
If reac_volume is bigger than the available stacks this will do nothing, as this condition will be true: https://github.com/Bubberstation/Bubberstation/blob/master/code/game/objects/items/stacks/stack.dm#L508

Reply:
Nevermind, the previous min ensures that can't happen. Still a bit sketchy, but this should work.

Reply:
I'll mark this one as resolved, then.